tests/ChatTests/Groups.hs

```diff
@@ -3262,24 +3262,24 @@ testGLinkApproveThenReviewMember =
       eve <## "bad chat command: not current member"
 
       -- pending member and moderators can send messages to each other
-      alice ##> "/_send #1 @support-5 text 11"
-      alice <# "#team <<support: eve>> 11"
-      [cath, dan] *<# "#team alice> <<support: eve>> 11"
-      eve <# "#team alice> <<support>> 11"
+      alice ##> "/_send #1(_support:5) text 11"
```

Comment:
why "_support" and not "support"?

Reply:
to differentiate from cli api that will use `(support: bob)`

Reply:
(to avoid ambiguity for names consisting of digits)